mkdocs.yml

```diff
@@ -42,6 +42,7 @@ nav:
       - OSS GPU: patterns/single-new-eks-observability-accelerators/single-new-eks-gpu-opensource-observability.md
       - OSS Java Mon : patterns/single-new-eks-observability-accelerators/single-new-eks-java-opensource-observability.md
       - OSS Nginx Mon : patterns/single-new-eks-observability-accelerators/single-new-eks-nginx-opensource-observability.md
+      - EKS Fargate OSS: patterns/single-new-eks-observability-accelerators/single-new-eks-fargate-opensource-observability.md 
```

Comment:
lets create a Separate folder for `EKS Fargate` and under that add this. Also move AWS Native Fargate pattern too.

Reply:
Not taken care

Reply:
done